### PR TITLE
improving code to be flake pep8 compliant.

### PR DESCRIPTION
**Pull request by eolivi**

### CODE REVIEW

File: corticalmapping_search.py [hunk 1]
```diff
@@ -49,52 +49,65 @@ def main(argv):
...
     ax.set_xlabel('alpha')
     ax.set_ylabel('beta')
     ax.set_zlabel('RDM + MAG')
     i = np.nonzero(zs == np.min(zs))
-    sys.stderr.write('xs = ' + str(xs[i]) + ' ys = ' + str(ys[i]) + ' rdm+mag= ' + str(np.min(zs)) +"\n")
-    sys.stdout.write('xs = ' + str(xs[i]) + ' ys = ' + str(ys[i]) + ' rdm+mag= ' + str(np.min(zs)) +"\n")
+    sys.stderr.write('xs = ' + str(xs[i]) + ' ys = ' + str(ys[i]) +
+                     ' rdm+mag= ' + str(np.min(zs)) + "\n")
+    sys.stdout.write('xs = ' + str(xs[i]) + ' ys = ' + str(ys[i]) +
+                     ' rdm+mag= ' + str(np.min(zs)) + "\n")
```
Comment (agramfort) [inline corticalmapping_search.py]: you should use:

```
 from __future__ import print_function
 print('spam', file=sys.stderr)
```

File: om_compare.py [hunk 2]
```diff
@@ -7,35 +7,39 @@
...
-def rdmmag(X1,X2):
+
+def rdmmag(X1, X2):
     X1 = vtk_to_numpy(X1)
     X2 = vtk_to_numpy(X2)
-    rdm  = la.norm(X1/la.norm(X1)-X2/la.norm(X2))
+    rdm = la.norm(X1 / la.norm(X1) - X2 / la.norm(X2))
     rmag = abs(1.-la.norm(X2)/la.norm(X1))
-    print "  RDM = ",rdm, "\t  rMAG = ",rmag
+    print "  RDM = ", rdm, "\t  rMAG = ", rmag
```
Comment (agramfort) [inline om_compare.py]: use print() for python 3

File: om_compare.py [hunk 3]
```diff
@@ -45,41 +49,54 @@ def compare_vtp(f1,f2):
...
 
     # TODO remove the min here for full comparisons
-    for i_s in range(min(nb_sources,5)):
+    for i_s in range(min(nb_sources, 5)):
         print'\033[91m' + "Source " + str(i_s) + " :" + '\033[0m'
-        V1 = poly1.GetPointData().GetArray('Potentials-'+str(i_s));
-        P1 = poly1.GetCellData().GetArray('Currents-'+str(i_s));
-        V2 = poly2.GetPointData().GetArray('Potentials-'+str(i_s));
-        P2 = poly2.GetCellData().GetArray('Currents-'+str(i_s));
+        V1 = poly1.GetPointData().GetArray('Potentials-'+str(i_s))
```
Comment (agramfort) [inline om_compare.py]: I prefer 
```
'Potentials-%s' % i_s
```

File: om_compare.py [hunk 4]
```diff
@@ -45,41 +49,54 @@ def compare_vtp(f1,f2):
...
+            conn2.Update()
+            V1 = (conn1.GetOutput().GetPointData()
+                  .GetArray('Potentials-'+str(i_s)))
+            P1 = (conn1.GetOutput().GetCellData()
+                  .GetArray('Currents-'+str(i_s)))
+            V2 = (conn2.GetOutput().GetPointData()
+                  .GetArray('Potentials-'+str(i_s)))
+            P2 = (conn2.GetOutput().GetCellData()
+                  .GetArray('Currents-'+str(i_s)))
             print " \t potentials :",
```
Comment (agramfort) [inline om_compare.py]: print( )

**Comment by agramfort** [comment]
is it possible to make a travis flake8 checker 

cf 

make flake

in mne-python code base.

it's best way to make sure we don't forget anything

File: om_compare.py (hunk 2 above)
Comment (eolivi) [reply om_compare.py]: right ! got it for prints

**Comment by agramfort** [comment]
now you need to rebase here :)

**Comment by eolivi** [comment]
done and green

**Comment by agramfort** [comment]
merged by rebase. See the last commit and you will see that there is a bit more work to make travis happy

thanks a lot @eolivi

**Comment by agramfort** [comment]
see https://github.com/openmeeg/openmeeg_sample_data/commit/f323c01ba6e858b35757bf980151f314e0b4bd42

**Pull request merged**